### DISCUSSION TITLE
Custom orientation

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/neper#526] [Q&A] [answered]
I used ATEX discrete EBSD data to get the grain orientation file, but the output file has four columns. The first three columns of data are grain orientation, but the fourth column of data is volume----It represents orientation weight. I want to use neper to create rve. When inputting custom orientation, if this volume needs to be taken into account, what should I do？
![F%4XS}RB~ 7NQEY{X52A42W](https://user-images.githubusercontent.com/71123582/180789303-82eb4068-8906-49f0-a950-55895ea6a2be.png)

**Comment by rquey** [comment, marked as answer]
It's a vague question, but I will assume that you want to create a statistically-equivalent RVE.

I think that *Volume* corresponds to the surface area of a grain on the EBSD map.  Note that, for a 3D polycrystal, it is not necessarily representative of the volume of the corresponding grain.

So, you could just disregard the volume data and use only the orientation.

If you want to use the volume data, targeting a given number of grains in the RVE, you could write a script to randomly pick orientations from the experimental data by taking into account the associated volume/weight (the higher the weight, the more likely the orientation to be picked).

**Reply by Ming-is** [reply]
Sorry, I don't quite understand the method you mentioned. I asked the developers of ATEX about the volume here, saying it represents the orientation weight, so my plan is to divide them by the maximum value to normalize them and assign values. I found that damask has similar options to assign crystal weight, so I'm curious whether neper has similar options?
